app/views/games/show.html.erb

```diff
@@ -1,99 +1,155 @@
+          <%= content_tag(:div, 'Black Pawn', class: 'chess-piece pawn-black text-hide', data: { pieceid: @pieces[20].id }) %>
+        </td>
+        <td class="chess-dark-square" data-columnindex="5">
+          <%= content_tag(:div, 'Black Pawn', class: 'chess-piece pawn-black text-hide', data: { pieceid: @pieces[21].id }) %>
+        </td>
+        <td class="chess-light-square" data-columnindex="6">
+          <%= content_tag(:div, 'Black Pawn', class: 'chess-piece pawn-black text-hide', data: { pieceid: @pieces[22].id }) %>
+        </td>
+        <td class="chess-dark-square" data-columnindex="7">
+          <%= content_tag(:div, 'Black Pawn', class: 'chess-piece pawn-black text-hide', data: { pieceid: @pieces[23].id }) %>
+        </td>
       </tr>
-      <tr>
-        <td class="chess-dark-square" id="row0col7"></td>
-        <td class="chess-light-square" id="row0col6"></td>
-        <td class="chess-dark-square" id="row0col5"></td>
-        <td class="chess-light-square" id="row0col4"></td>
-        <td class="chess-dark-square" id="row0col3"></td>
-        <td class="chess-light-square" id="row0col2"></td>
-        <td class="chess-dark-square" id="row0col1"></td>
-        <td class="chess-light-square" id="row0col0"></td>
+      <tr data-rowindex="7">
+        <td class="chess-dark-square" data-columnindex="0">
+          <%= content_tag(:div, 'Black Rook', class: 'chess-piece rook-black text-hide', data: { pieceid: @pieces[24].id }) %>
+        </td>
+        <td class="chess-light-square" data-column-index="1">
+          <%= content_tag(:div, 'Black Knight', class: 'chess-piece knight-black text-hide', data: { pieceid: @pieces[26].id }) %>
+        </td>
+        <td class="chess-dark-square" data-column-index="2">
+          <%= content_tag(:div, 'Black Bishop', class: 'chess-piece bishop-black text-hide', data: { pieceid: @pieces[28].id }) %>
+        </td>
+        <td class="chess-light-square" data-column-index="3">
+          <%= content_tag(:div, 'Black King', class: 'chess-piece king-black text-hide', data: { pieceid: @pieces[31].id }) %>
+        </td>
+        <td class="chess-dark-square" data-column-index="4">
+          <%= content_tag(:div, 'Black Queen', class: 'chess-piece queen-black text-hide', data: { pieceid: @pieces[30].id }) %>
+        </td>
+        <td class="chess-light-square" data-column-index="5">
+          <%= content_tag(:div, 'Black Bishop', class: 'chess-piece bishop-black text-hide', data: { pieceid: @pieces[29].id }) %>
+        </td>
+        <td class="chess-dark-square" data-column-index="6">
+          <%= content_tag(:div, 'Black Knight', class: 'chess-piece knight-black text-hide', data: { pieceid: @pieces[27].id }) %>
+        </td>
+        <td class="chess-light-square" data-column-index="7">
+          <%= content_tag(:div, 'Black Rook', class: 'chess-piece rook-black text-hide', data: { pieceid: @pieces[25].id }) %>
+        </td>
```

Comment:
What happens if I refresh the page? This looks like it wouldn't render pieces in the middle of the board.